instances/azp-small-build-asg/init.sh.tpl

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env bash
+
+set -e
+
+function terminate {
+    # Terminate instances in 1 min
+    shutdown -h +1
+}
+trap terminate EXIT
+
+
+# shellcheck source=ami-build/scripts/run-fun.sh
+. /usr/local/share/run-fun.sh
+
+AWS_DEFAULT_REGION="$(aws_get_region)"
+export AWS_DEFAULT_REGION
+
+
+# shellcheck disable=SC2154
+agent_start_build \
+    "${azp_pool_name}" \
+    "${asg_name}" \
+    "${instance_profile_arn}" \
+    "${role_name}" \
+    ${CACHE_BUCKET} \
+    ${CACHE_PREFIX} \
+
+# figure out what the CACHE_ vars are
```

Comment:
Did you intend to remove this comment before we merge it in?

Reply:
ah yep - now i rem why it was WIP - this needs to be resolved i think